photon-client/src/stores/settings/GeneralSettingsStore.ts

```diff
@@ -28,10 +29,12 @@ export const useSettingsStore = defineStore("settings", {
         network: {
             ntServerAddress: "",
             shouldManage: true,
+            canManage: true,
             connectionType: NetworkConnectionType.DHCP,
             staticIp: "",
             hostname: "photonvision",
-            runNTServer: false
+            runNTServer: false,
+            networkInterfaceNames: []
```

Comment:
Proper placeholder values are required, we can't just assume an empty string. (Pipeline settings are valid; other Network settings are valid)

Reply:
That's an empty list of strings, no?

Reply:
We should have some default values that are probable. It is better than just having an empty field which would leave people confused in the event of a no connect

Reply:
Sure. Though this should probably get hidden if the list length is zero?